### ISSUE TITLE
Title

### ISSUE THREAD

**Issue by jeromekelleher** [sgkit-dev/vcf-zarr-publication#87]
Currently the title is "Columnar VCF for Biobank scale datasets using Zarr" which isn't great. Let's get a better title.

**Comment by jeromekelleher** [comment]
Some possibilities:

- Analysis ready VCF (A nod to Ryan Abernathy's narrative for [scientific data in the cloud](https://medium.com/pangeo/the-case-for-cloud-in-science-3f98f6538a33))
- Efficiently storing and processing Biobank-scale VCF with Zarr
- Others? (Hoping people will jump in here with some ideas...)

**Comment by tomwhite** [comment]
I like the "Analysis ready" part, so maybe "Analysis ready VCF for Biobank scale datasets using Zarr"

**Comment by jeromekelleher** [comment]
I like it, but I feel it's a bit wordy and there's too many clauses. How about

> Analysis ready VCF at Biobank scale using Zarr

**Comment by jeromekelleher** [comment]
I'm considering just "Analysis-ready VCF at Biobank scale", but I think it's good to keep Zarr in the title. Happy to discuss further is someone has better ideas.
